roadmap/implementers-guide/src/types/candidate.md

```diff
@@ -64,7 +72,7 @@ This struct is pure description of the candidate, in a lightweight format.
 /// A unique descriptor of the candidate receipt.
 struct CandidateDescriptor {
 	/// The ID of the para this is a candidate for.
-	para_id: Id,
+	para_id: ParaId,
```

Comment:
```suggestion
	para: ParaId,
```
consistency

Reply:
(have altered for consistency in the other direction)